packages/xod-cli/src/constants.js

```diff
@@ -0,0 +1,29 @@
+import path from 'path';
+
+export const BUNDLED_WORKSPACE_PATH = path.resolve(
+  __dirname,
+  '..',
+  'bundle',
+  'workspace'
+);
+
+export const BUNDLED_TABTEST_WORKSPACE_PATH = path.resolve(
+  __dirname,
+  '..',
+  'bundle',
+  'tabtest-workspace'
+);
+
+export const BUNDLED_TABTEST_SRC_PATH = path.resolve(
+  __dirname,
+  '..',
+  'bundle',
+  'tabtest-cpp'
+);
+
+export const BUNDLED_CATCH2_PATH = path.resolve(
+  __dirname,
```

Comment:
These are not constants. `3.141` and `"hello"` are constants, these are not and will shoot your leg once upon a time. Suggest converting to functions (`resolveBundledCatch2Path`, etc).… And rename the module.